starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,55 @@
+* {
+  margin: 0;
```

Comment:
Reseting default CSS rules is a good practice!
Some people actually took the time to make pretty good "reset blocks" ready for you to copy paste at the beginning of your projects, you can find a nice one here: https://meyerweb.com/eric/tools/css/reset/